### PR TITLE
MNT: Change naming to MET as its plotting met data.

### LINKED ISSUE #652: Example Incorrect
### Description

This example says it's for plotting sounding data but the example uses MET data.  We should update this to be what it was originally intended.
https://arm-doe.github.io/ACT/source/auto_examples/plotting/plot_sonde.html

### PR DESCRIPTION
**Pull request by zssherman**
- [x] Closes #652 
- [x] Documentation reflects changes
- [x] PEP8 Standards or use of linter

### CODE REVIEW

File: examples/plotting/plot_met.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
 """
-Plot a timeseries of sounding data
+Plot a timeseries of MET data
 ----------------------------------------------------
 
-This is a simple example for how to plot a timeseries of sounding
+This is a simple example for how to plot a timeseries of MET
```
Comment (mgrover1) [inline examples/plotting/plot_met.py:5]:
```suggestion
This is a simple example for how to plot a timeseries of surface meteorology (MET) data
```

**Review by mgrover1 — CHANGES_REQUESTED** [review]
One small suggestion

**Comment by zssherman** [comment]
Ooo yeah good point! Will fix that now

**Comment by zssherman** [comment]
@rcjackson Did pysp2 update anything that would change the waveform stats calculation?

**Comment by zssherman** [comment]
Seeing some unit changes etc recently to it.

**Comment by AdamTheisen** [comment]
@zssherman if we are changing this to just be a simple plot of the time series data, we can remove the LCL calculations from this example.  I was somewhat thinking of this the other way around in that we should update the example to plot the SONDE time series with the LCL plotted in a plot of the altitude or something like that.  I think that is what @rcjackson was maybe intending to do with this one.  Thoughts?

**Comment by zssherman** [comment]
@AdamTheisen Ah makes sense! I can switch things around to be what was originally intended!

**Comment by zssherman** [comment]
@AdamTheisen Example has been updated for sonde data. Updating the unit tests now to match the pysp2 changes that are failing the CI.

**Comment by zssherman** [comment]
@mgrover1 Done

**Comment by zssherman** [comment]
Confused on this class init failures for only python 3.8:
TypeError: __init__() got an unexpected keyword argument 'base'

**Comment by zssherman** [comment]
Ah so all tests failing for that base error are all older xarray version its looks like using base parameter in resample.

**Comment by zssherman** [comment]
@AdamTheisen @mgrover1 Do we drop 3.8 as we are three ahead with 3.9, 3.10 and 3.11 or do we find a work around for this base error in resampling.

**Comment by mgrover1** [comment]
Let's pin pandas<2.0... it is an issue on the xarray/pandas/flox side. We should consider pinning version in tests/such

**Comment by zssherman** [comment]
@mgrover1 Sounds like a plan!

**Comment by zssherman** [comment]
@mgrover1 I can't remember if that syntax is right for requirements.txt. I know it is for the .yml. Also two numpy installs removed one of them.

**Comment by zssherman** [comment]
And do you have the issue or where you found the info on the pandas/flox issue.

**Comment by mgrover1** [comment]
https://github.com/pydata/xarray/issues/7716

**Comment by zssherman** [comment]
@mgrover1 Cool seems to be working! Just the usual stalled tests.

**Comment by zssherman** [comment]
@AdamTheisen @mgrover1 Once you both think this is good to merge, I can go do that

**Review by mgrover1 — APPROVED** [review]

**Review by AdamTheisen — APPROVED** [review]

**Pull request merged**